### PR TITLE
Add tests for `select` behavior when given no options

### PR DESCRIPTION
**Pull request by colinking**
Follow-up to https://github.com/airplanedev/js-sdk/pull/147

The change in that PR didn't work. This PR fixes that and adds tests.

### CODE REVIEW

**Comment by colinking** [comment]
Current dependencies on/for this PR:
* main
  * **PR #148** <a href="https://app.graphite.dev/github/pr/airplanedev/node-sdk/148" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/airplanedev/node-sdk/148?utm_source=stack-comment).

File: packages/airplane/internal/runtime/mock.ts [hunk 1]
```diff
@@ -0,0 +1,70 @@
+import { ClientOptions } from "../api/client";
+import { DisplayConfig, ParamSchema, ParamValues, Run, Session } from "../api/types";
+import { RuntimeInterface } from "./index";
+
+export const createMockRuntime = (runtime: Partial<RuntimeInterface>): RuntimeInterface => {
```
Comment (calufornia) [inline packages/airplane/internal/runtime/mock.ts:5]: Also add `"mock"` as a valid runtime [here](https://github.com/airplanedev/js-sdk/blob/bae4efe01943c71dc712e83d1bdc2bc41b3f2642/packages/airplane/internal/runtime/index.ts#L40) - up to you, it's just for tests.

File: packages/airplane/internal/prompt/index.test.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      return select("Test Parameter", [], {
+        // The default is not one of the valid options.
+        default: "test",
+      });
+    }).rejects.toThrowErrorMatchingInlineSnapshot(
+      `"The required prompt parameter, "Test Parameter", has no options to select from."`
+    );
+  });
+
+  test("succeeds with no options when optional", async () => {
```
Comment (calufornia) [inline packages/airplane/internal/prompt/index.test.ts:37]: Could you add a test with a required prompt parameter that succeeds?

**Review by calufornia — APPROVED** [review]

File: packages/airplane/internal/runtime/mock.ts (hunk 1 above)
Comment (colinking) [reply packages/airplane/internal/runtime/mock.ts:5]: It does seem a bit odd for a plugins system to hardcode the list of plugins. Maybe we remove `RuntimeKind`? For now, I'm leaving the type cast of `"mock" as RuntimeKind` as a workaround.

**Pull request merged**